nixos/modules/installer/tools/nixos-rebuild.sh

```diff
@@ -227,6 +227,13 @@ if [ -z "$_NIXOS_REBUILD_REEXEC" -a -n "$canRun" -a -z "$fast" ]; then
     fi
 fi
 
+# Find configuration.nix and open editor instead of building.
+if [ "$action" = edit ]; then
+    NIXOS_CONFIG=${NIXOS_CONFIG:-$(nix-instantiate --eval --strict -E '<nixos-config>')}
```

Comment:
Same as `nix-instantiate --find-file nixos-config`

Reply:
Didn't know that was exposed on the cli.